pkg/file/default_install_scripts.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pkg": "installer -pkg $INSTALLER_PATH -target /",
```

Comment:
This might be an artifact from the new story and not necessarily needed for this bugfix no?

Reply:
thanks!